doc-site/docs/guides/kubernetes-cluster.md

```diff
@@ -1,3 +1,6 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
```

Comment:
looks like the sidebar actually requires the md metadata to be on top of the import

```
---
title: Operating your Kubernetes Cluster
sidebar_label: Kubernetes Cluster operations
sidebar_position: 2
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
```